eyeshade/controllers/owners.js

```diff
@@ -51,6 +55,7 @@ v1.getWallet = {
       } catch (ex) {
         debug('status', { reason: ex.toString(), stack: ex.stack })
         runtime.captureException(ex, { req: request, extra: { owner: owner } })
+        return reply(boom.boomify(ex))
```

Comment:
I don't think we want to return error here

Reply:
oh, this is just so that the subsequent lines are not hit, so that we don't get the reply called twice error from hapi. it doesn't return the error per-se, but the reply interface from hapi.

Reply:
we weren't calling reply previously in this block though

Reply:
right, we weren't calling it, so if there was a problem anywhere it was with the `wallet` not existing, and skipping the block in the try or via an error (say, from sentry) that was reported to sentry and then continuing on with the handler, returning the rates object only

Reply:
chatted out of band - need to check that expired token does not cause an exception